### PR TITLE
fix: typo in destroy method

### PR DESCRIPTION
**Pull request by suyashpatil78**
…eadability

### <samp>🤖 Generated by Copilot at 82b856d</samp>

This pull request fixes typos in the method names related to the FreshChat widget integration in `app.component.ts` and `fresh-chat.service.ts`. It also renames some parameters in the `FreshChatService` class to improve readability.

### <samp>🤖 Generated by Copilot at 82b856d</samp>

> _There once was a method called `destory`_
> _That was meant to remove a chat toy_
> _But the name was a typo_
> _So they fixed it with gusto_
> _And renamed some params to avoid cloy_

### <samp>🤖 Generated by Copilot at 82b856d</samp>

* Fix typo in `destroy` method name in `AppComponent` and `FreshChatService` classes ([link](https://github.com/fylein/fyle-mobile-app/pull/2150/files?diff=unified&w=0#diff-884f7f49640e5923f6bcac4c51d90340330a178f662defbe61e5f5aac1c512deL194-R194), [link](https://github.com/fylein/fyle-mobile-app/pull/2150/files?diff=unified&w=0#diff-5cd10d8760f5baff09c3d8e6d24919c3070e28033e2eb0afdf12995afd1c6300L47-R49))
* Rename parameters `i` and `t` to `document` and `elementId` in `initialize` method of `FreshChatService` class for better readability and clarity ([link](https://github.com/fylein/fyle-mobile-app/pull/2150/files?diff=unified&w=0#diff-5cd10d8760f5baff09c3d8e6d24919c3070e28033e2eb0afdf12995afd1c6300L129-R139))

## Clickup
app.clickup.com

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/core/services/fresh-chat.service.ts [hunk 1]
```diff
@@ -126,17 +126,17 @@ export class FreshChatService {
     });
   }
 
-  private initialize(i, t) {
+  private initialize(document, elementId) {
     const that = this;
```
Comment (yksolanki9) [inline src/app/core/services/fresh-chat.service.ts:130]: Not needed, use this everywhere

File: src/app/core/services/fresh-chat.service.ts [hunk 2]
```diff
@@ -44,9 +44,9 @@ export class FreshChatService {
     return (window as any) && (window as any).fcWidget && (window as any).fcWidget.open();
   }
 
-  destory() {
-    if ((window as any) && (window as any).fcWidget && (window as any).fcWidget.destory) {
-      (window as any).fcWidget.destory();
+  destroy() {
+    if ((window as any) && (window as any).fcWidget && (window as any).fcWidget.destroy) {
```
Comment (yksolanki9) [inline src/app/core/services/fresh-chat.service.ts:48]: Remove type assertion

File: src/app/core/services/fresh-chat.service.ts [hunk 3]
```diff
@@ -126,17 +126,17 @@ export class FreshChatService {
     });
   }
 
-  private initialize(i, t) {
+  private initialize(document, elementId) {
```
Comment (yksolanki9) [inline src/app/core/services/fresh-chat.service.ts:129]: Missing types

**Review by yksolanki9 — CHANGES_REQUESTED** [review]

File: src/app/core/services/fresh-chat.service.ts [hunk 4]
```diff
@@ -19,45 +27,43 @@ export class FreshChatService {
...
+    concat(this.networkService.isOnline(), networkWatcherEmitter.asObservable()).subscribe(async (isOnline) => {
+      const eou = await this.authService.getEou();
       if (eou && isOnline) {
-        const orgUserSettings = await that.getOrgUserSettings();
+        const orgUserSettings = await this.getOrgUserSettings();
         if (
           orgUserSettings &&
           orgUserSettings.in_app_chat_settings &&
           orgUserSettings.in_app_chat_settings.allowed &&
           orgUserSettings.in_app_chat_settings.enabled
```
Comment (yksolanki9) [inline src/app/core/services/fresh-chat.service.ts:41]: Replace with optional chaining

**Review by yksolanki9 — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]
fix existing comment, rest lgtm

**Pull request merged**